beluga_amcl/test/cmake/ament.cmake

```diff
@@ -14,6 +14,20 @@
 
 find_package(ament_cmake_gmock REQUIRED)
 
+file(COPY test_data DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
@serraramiro1 nit: did you consider passing the path to test data as a compiler definition?

Reply:
I didn't know that was a thing, tbh. 
I don't see a lot of value on changing this and this is the way it's being done in `beluga`, so I'll keep it as-is.

Reply:
I didn't know that was a thing, tbh. 
I don't see a lot of value on changing this and this is the way it's being done in `beluga`, so I'll keep it as-is.